### DISCUSSION TITLE
Architectural Decision Records

### DISCUSSION THREAD

**Discussion by Andygol** [k3ilona/publisher-slack-bot#3] [General]
> **Файл ADR** (Architectural Decision Records) — обґрунтований вибір дизайну програмного забезпечення, який відповідає функціональним або нефункціональним вимогам, що є важливими з архітектурної точки зору.

Це має бути щось подібне до того що ми вже робили на 4-му тижні – https://github.com/Andygol/AsciiArtify/blob/main/doc/Concept.md

**Comment by vit-um** [comment]
Закинув в репо чернетку цього документу: https://github.com/k3ilona/demo-repository/blob/main/doc/Concept.md
Потрібно попрацювати над текстом.

**Reply by Andygol** [reply]
Не те репо 😏

**Reply by vit-um** [reply]
Перенесу пізніше в це репо. Але не зовсім тоді зрозумів призначення `demo-repository`. Чому воно демо?

**Reply by Andygol** [reply]
Це репо створено автоматично GitHub під час реєстрації організації.

**Reply by vit-um** [reply]
Було б добре розуміти призанчення цього репо, що створено автоматично, та чи будемо ми над ним якось працювати, та чи можна його видалити взагалі. Наскільки я бачу воно дозволяє швидко розгорнути веб-сайт нашої команди. 
Пепеніс док в гілку дев правильного репо, очикую прийняття пул реквесту.

**Reply by Andygol** [reply]
Це звичайне демонстраційне репо для того, щоб показати можливості GitHub. Воно жодним чином не впливає на розробку бота
